src/charactersheet/viewmodels/common/exhibit/index.js

```diff
@@ -0,0 +1,61 @@
+import autoBind from 'auto-bind';
+import { get } from 'lodash';
+import { CoreManager, Notifications } from 'charactersheet/utilities';
+import { ViewModel } from 'charactersheet/viewmodels/abstract';
+import { PartyService } from 'charactersheet/services';
+import { observable, components, pureComputed } from 'knockout';
+import template from './index.html';
+
+
+export class ExhibitViewModel extends ViewModel {
+
+    constructor() {
+        super();
+        autoBind(this);
+
+        this.party = observable(PartyService.party);
+        this.playerType = observable(CoreManager.activeCore().type.name());
+        this.fullScreen = observable(false);
+    }
+
+    async setUpSubscriptions() {
+        super.setUpSubscriptions();
+
+        this.subscriptions.push(Notifications.party.changed.add(this.partyDidChange));
+    }
+
+    // UI
+
+    isConnectedToParty = pureComputed(() => (!!this.party()));
+
+    imageUrl = pureComputed(() => (
+        get(this.party(), 'exhibit.imageUrl', null)

```

Comment:
will an `exhibit` object always be in the response? not sure how lodash handles the `.imageUrl` if `exhibit` is null

Reply:
Get is a best-effort attempt. If either `party`, `exhibit`, or `imageUrl` is not found, then the result is instead the default `null`.